scaleway/ultraviolet
# refactor(Tabs): use menu v2

## Summary

## Type

- Refactor

### Summarise concisely:

#### What is expected?

Migration of Menu to MenuV2 into Tabs component.

#### How to test?

1. Go in deployed storybook version and go to Tabs component

2. Check that the tabs with "More" menu works especially stories "With Menu" and "Show Case"

3. You can also checkout this branch locally, build it and test it on other projects that uses ultraviolet ui
